### ISSUE TITLE
Loss function plot

### ISSUE THREAD

**Issue by hjh17** [paaalli/tf-experiments#2]
I would like to explore the results from training visually. 

After each iteration through the whole training data (all the mini batches), the performance should be measured on

- the **training** set
- the **dev** set

This allows us to gather information on over/underfitting of the model on the data.

http://cs231n.github.io/assets/nn3/learningrates.jpeg

Later on, we can connect this feature with [Tensorboard](https://www.tensorflow.org/get_started/summaries_and_tensorboard)...
